subsys/dsp/Kconfig

```diff
@@ -32,6 +39,17 @@ config DSP_BACKEND_CUSTOM
 	  Rely on the application to provide a custom DSP backend. The implementation should be
 	  added to the 'zdsp' build target by the application or one of its modules.
 
+config DSP_BACKEND_ARCMWDT
+	bool "Use the mwdt library as the math backend"
+	depends on ARCMWDT_LIBC
+	depends on CMSIS_DSP
```

Comment:
I just noticed you have this `depends on CMSIS_DSP` here, on line 24 we select the CMSIS backend if we have the `CMSIS_DSP` selected. so that means that this backend will never be selected by default. Do we really need this dependency?

Reply:
Synopsys DSP library currently doesn't have corresponding functions like clip(), and(), or(), not(), xor() comparing to CMSIS DSP library. These functions are mapped to CMSIS library temporarily. I think it is needed to inform users to arcmwdt backend is partly dependent on cmsis_dsp.